astrobee/launch/sim.launch.py

```diff
@@ -132,6 +132,21 @@ def generate_launch_description():
         #   <!-- Connect and update environment variables if required -->
         #   <machine unless="$(eval arg('sim')=='local')" name="sim_server" default="true"
         #            address="$(arg sim)" user="astrobee" password="astrobee" timeout="10"/>
+        # Update the environment variables relating to absolute paths
+        SetEnvironmentVariable(name="ASTROBEE_CONFIG_DIR",
+                               value=os.getenv("ASTROBEE_CONFIG_DIR", "/home/astrobee/native/config"),
+                               condition=LaunchConfigurationNotEquals("sim", "local")),
```

Comment:
I think this is correct but I'm confused. So you are only setting the environment variable if sim doesn't equal local. Do we not need it set when the sim is local? Or are we requiring users to set it? Also, where did that default path come from? Does ROS2 not have a find package anymore?

Reply:
So that is me converting this from astrobee.launch, line 130 to ROS2:
`<env unless="$(eval arg('sim')=='local')"`
if the sim is local it will use the:
`SetEnvironmentVariable(name="ASTROBEE_CONFIG_DIR",    value=os.getenv("ASTROBEE_CONFIG_DIR",    get_path("config"))),`
that was defined in this file previously

The find package now changed to be the function `get_package_share_directory(package)`. This is now the only thing available to my knowledge and points to the install folder, not build or src folder, so there is the added nuance that everything needs to be installed when launching

Reply:
Also I'm not sure where those paths come from, they pre-date me xD

Reply:
Yeah, I was just confused. Thanks for clarifying.